FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Common/Models/Jobs/Job.cs

```diff
@@ -44,9 +47,34 @@ public class Job
             {
                 _ = ResourceProgresses.TryAdd(resource, null);
                 _ = TotalResourceCounts.TryAdd(resource, 0);
-                _ = ProcessedResourceCounts.TryAdd(resource, 0);
-                _ = SkippedResourceCounts.TryAdd(resource, 0);
-                _ = PartIds.TryAdd(resource, 0);
+
+                // Temporarily set schema type list only contains single value for each resource type
+                // E.g:
+                // Schema list for "Patient" resource is ["Patient"]
+                // Schema list for "Observation" resource is ["Observation"]
+                _ = SchemaTypeMap.TryAdd(resource, new List<string>() { resource });
+
+                // After supporting customized schema, the schema type map will be set below when customized schema is enable
+                // _ = SchemaTypesMap.TryAdd(resource, new List<string>() { resource, $"{resource}_customized"});
```

Comment:
Will we change the current default schema to use same Json schema file just like the custom schema files?